.storybook/FormField.js

```diff
@@ -179,6 +188,72 @@ storiesOf('InputField', module)
       </Flex>
     ))
   )
+  .add(
+    'FormField composed with Select',
+    withInfo(
+      'This example demonstrates how the input field will behave with a select dropdown'
+    )(() => (
+      <Flex width={900}>
+        <Box width={1 / 3} pl={3}>
+          <Text bold pt={4} pb={2}>
+            User Types a Value
+          </Text>
+          <FormField onChange={() => {}}>
```

Comment:
Instead of empty function in `onChange`, perhaps the PR can take on the **Storybook Addon Actions** to "_hook up_" with the `onChange` synthetic event, such as:
```
action('onChange event is triggered in FormField')
```
https://github.com/storybooks/storybook/tree/master/addons/actions#storybook-addon-actions

Reply:
Personally, I'd rather avoid custom storybook APIs as much as possible and try to keep our examples decoupled from that sort of thing

Reply:
I updated my test, as well as the others in the file.